src/components/Letter.tsx

```diff
@@ -302,7 +272,7 @@ export function LetterView({
             <div className="stamp cursor-pointer">
               <label className="flex justify-center w-full h-32 px-4 appearance-none cursor-pointer hover:background-gray-400 focus:outline-none">
                 {fromStamp ? (
-                  <img src={fromStamp} />
+                  <Stamp person={from} />
```

Comment:
err wait this is going to break the letterform, which relies on `fromStamp` being set that is not from a person

Reply:
see line 209 here